.gitmodules

```diff
@@ -0,0 +1,5 @@
+[submodule "tests/acceptance/image-tests"]
+	path = tests/acceptance/image-tests
```

Comment:
Setting such path implies changes in _mender-qa/scripts/jenkins-yoctobuild-build.sh_. Cannot we use simply _tests/acceptance_ path here? Having tests in two paths would cause the py.test must be run twice, wouldn't it?

Reply:
Ok, I saw [mender-qa/pull/217](https://github.com/mendersoftware/mender-qa/pull/217) So only question is whether py.test searches tests recursively, as mutual tests for Yocto and non-Yocto builds are stored now in _meta-mender/tests/acceptance/image-tests/tests_?

Reply:
Yes, it searches recursively, I checked the log to be sure, and it runs all tests including those. The directory nesting is a bit cumbersome, but a submodule must be in a subdirectory, so we have no choice.